pkg/network/aci_network.go

```diff
@@ -280,7 +281,6 @@ func (pn *ProviderNetwork) CreateOrUpdateACISubnet(ctx context.Context, subnetsC
 
 	// only set the address prefix and prefixes if we are creating a new subnet
 	if isCreate {
-		subnet.Properties.AddressPrefix = &pn.SubnetCIDR
```

Comment:
why do we need to remove it in the isCreate case?

Reply:
Since Properties.AddressPrefix is required in the PUT payload, I am moving it out of just the create block and moving it up for both create and update scenarios